packages/terra-tag/tests/wdio/tag-spec.js

```diff
@@ -1,28 +1,29 @@
 Terra.describeViewports('Tag', ['tiny'], () => {
-  describe('Default', () => {
-    before(() => browser.url('/#/raw/tests/terra-tag/tag/default-tag'));
-    Terra.it.validatesElement();
+  it('displays Default tag', () => {
+    browser.url('/raw/tests/terra-tag/tag/default-tag');
+    Terra.validates.element('default');
   });
 
-  describe('Href Tag', () => {
-    before(() => browser.url('/#/raw/tests/terra-tag/tag/href-tag'));
-    Terra.it.matchesScreenshot();
+  it('displays Href Tag', () => {
+    browser.url('/raw/tests/terra-tag/tag/href-tag');
+    Terra.validates.element('href');
   });
 
-  describe('Icon and Text Tag', () => {
-    before(() => browser.url('/#/raw/tests/terra-tag/tag/icon-and-text-tag'));
-    Terra.it.validatesElement();
+  it('displays Icon and Text Tag', () => {
+    browser.url('/raw/tests/terra-tag/tag/icon-and-text-tag');
+    Terra.validates.element('icon and text');
   });
 
-  describe('Icon and Text Long Tag', () => {
-    before(() => browser.url('/#/raw/tests/terra-tag/tag/long-text-tag'));
-    Terra.it.validatesElement();
+  it('displays Icon and Text Long Tag', () => {
+    browser.url('/raw/tests/terra-tag/tag/long-text-tag');
+    Terra.validates.element('icon and long text');
   });
 
   describe('OnClick Tag', () => {
-    before(() => browser.url('/#/raw/tests/terra-tag/tag/on-click-tag'));
-
-    Terra.it.validatesElement();
+    it('displays Default tag', () => {
```

Comment:
The case sensitivity on each of these `it` descriptions are inconsistent. This doesn't effect the test run but it's something to keep in mind as we're making these updates